schemas/2025-03-01/deploymentParameters.json

```diff
@@ -0,0 +1,102 @@
+﻿{
+  "id": "https://schema.management.azure.com/schemas/2025-03-01/deploymentParameters.json#",
```

Comment:
I know this is probably not the intended usage, but we typically add to the existing schemas rather than add new ones. (assuming we're not making breaking changes, which we generally would not do anyway).

The problem is that we have no mechanism to actually point people to use new schemas when they are published.